### PR TITLE
#161254612 Integrate Travis to run tests

### PR DESCRIPTION
**Pull request by evamaina**
#### What does this PR do?
```Integrate Travis with readme badge to run tests ```
```Integrate coveralls with readme badge```
#### Description of Task to be completed?
```Add .travis.yml file ```
```Add .coveralls.yml file ```
#### How should this be manually tested?
```https://github.com/andela/ah-orcas-frontend ```
```switch to ch-integrate-travis-161254612 branch```
```Check the README file```
#### Any background context you want to provide?
- Once this branch is merged to develop, it will build from develop branch as set up in the readme badge.The build should pass as I have already tested it in my current branch.See the attached screenshot.
- Travis makes working in a team for a software project easier with automated builds which are triggered automatically when each developer checks in their code to the repository. 
- It also helps to automatically run tests and other preliminary build steps

#### What are the relevant pivotal tracker stories?
[#161254612](https://www.pivotaltracker.com/story/show/161254612)
#### Screenshots 
image

#### Checklist:

- [x] My code follows the style guidelines of this project
- [x] At least 2 people have reviewed my PR
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] My PR has one commit.

### CODE REVIEW

File: .coveralls.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
+repo_token: DoePYPk8rkpl04RMeCMyPx5sGAY8Wo1Hf
```
Comment (TheSteelGuy) [inline .coveralls.yml]: append new line here

Comment (TheSteelGuy) [reply .coveralls.yml]: Everything else is okay

**Review by TheSteelGuy — APPROVED** [review]
LGTM

**Review by wrotich — APPROVED** [review]
LGTM

**Review by Eyansky — APPROVED** [review]
LGTM

**Pull request merged**